src/text/grapheme.rs

```diff
@@ -1,4 +1,4 @@
-use crate::prelude::*;
+use crate::{prelude::*, style::Styled};
```

Comment:
My preference is opposite this. Use an import even when it's a single import, because this leads to generally more readable and succinct code when it's used.  When code is dominated with information about where the items come from, the module path makes it more difficult to quickly scan the words.